GlyssenEngine/GlyssenEngine.csproj

```diff
@@ -39,19 +45,32 @@
     <None Remove="Resources\RelatedCharacters.xml" />
     <None Remove="Resources\SampleMRK.xml" />
     <None Remove="Resources\Silent.wav" />
+    <None Include="..\LICENSE">
+      <Pack>True</Pack>
+      <PackagePath></PackagePath>
+    </None>
   </ItemGroup>
 
   <ItemGroup>
     <PackageReference Include="DotNetZip" Version="1.13.4" />
+    <PackageReference Include="GitVersionTask" Version="5.1.3">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
```

Comment:
I guess VS or whatever tool you used added this line? It's not necessary, but if a tool put it there I'd leave it.

Reply:
Yup